code/game/objects/items/devices/traitordevices.dm

```diff
@@ -404,3 +404,26 @@
 
 	times_used = max_uses - 1
 	activate_batterer()
+
+/obj/item/handheld_mirror
+	name = "hand mirror"
+	desc = "Style, on the go!"
+	icon = 'icons/obj/device.dmi'
+	icon_state = "hand_mirror"
+	w_class = WEIGHT_CLASS_TINY
+	var/datum/ui_module/appearance_changer/appearance_changer_holder
+
+/datum/datum/ui_module_appearance_changer/appearance_changer_holder/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.hands_state)
```

Comment:
I'm a little confused by what this proc is supposed to be doing, exactly. Regardless, I think you probably don't want the double datum here?
```suggestion
/datum/ui_module_appearance_changer/appearance_changer_holder/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.hands_state)
```

Reply:
That is me being an absolute caveman learning to use tgui, will fix next commit

Reply:
Cool, looks a lot better.